api/environments/identities/tests/test_models.py

```diff
@@ -47,6 +51,25 @@ def test_create_identity_should_assign_relevant_attributes(self):
         assert isinstance(identity.environment, Environment)
         assert hasattr(identity, "created_date")
 
+    @mock.patch("environments.identities.models.dynamo_identity_table")
+    def test_create_identity_should_call_put_item(self, dynamo_identity_table):
+        # Given
+        dynamo_identity_table.put_item = mock.MagicMock()
```

Comment:
I don't think you need this? By default, any attribute of a MagicMock is also a MagicMock itself.